src/components/dynamic/admin/dashboards/Toolbar.jsx

```diff
@@ -28,6 +28,19 @@ const Toolbar = ({
   const [toggle, setToggle] = useState(false);
 
   const onClick = (value) => {
+    const selectedWinners = objects.some(
+      (obj) => obj.selected && obj.status === "winner"
+    );
+    if (selectedWinners) {
+      setPopup({
+        title: "Status Change Restricted",
+        text: "Changing status from is restricted. You can check the Prize page for more information.",
```

Comment:
Changing status from winner is restricted. Please remove the prize for this team before changing their status

Reply:
there should be a link option that u need to enable? so that the button takes u to the prizes page